docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -213,6 +213,9 @@ If you only need create or update commands, use the [az deployment group create]
 
 If generic update arguments and [az resource](../latest/docs-ref-autogen/resource.yml) don't meet your needs, you can use [az rest](/cli/azure/reference-index#az-res) command to call the REST API. The command automatically authenticates using the logged-in credential and sets header `Content-Type: application/json`. For more information, see [Azure REST API reference](/rest/api/azure/).
 
+> [!NOTE]
+> Calling service API directly with `az rest` is only a supplemental feature of Azure CLI. It only works if the REST API is invoked correctly and it is subject to change, for reasons including but not limited to Azure CLI permission change, Microsoft Graph service limitation, AAD Conditional Access policies, etc. For service REST API questions or issues, please [create a support ticket](/support/create-ticket/).
```

Comment:
```suggestion
> Calling a service API directly with `az rest` is a _supplemental_ feature of Azure CLI, and only works if the REST API is invoked correctly. These APIs are subject to change for reasons including, but not limited to, Azure CLI permission changes, Microsoft Graph service limitations, AAD Conditional Access policies, etc. For service REST API questions or issues, please [create a support ticket](/support/create-ticket/).
```

Reply:
Actually, I meant to say "Calling these APIs with `az rest` is subject to change".